sunscreen_math/src/metal_impl/mod.rs

```diff
@@ -24,13 +26,21 @@ const SHADERLIB: &[u8] = include_bytes!(concat!(
 const SHADERLIB: &[u8] =
     include_bytes!(concat!(env!("OUT_DIR"), "/curve25519-dalek.test.metallib"));
 
+pub struct Grid([(u64, u64); 3]);
```

Comment:
I think this is currently just adding `.0` noise, and probably a type synonym would be fine, but I suppose it doesn't hurt much. Could be useful in the future.